blocks/article-hero/article-hero.css

```diff
@@ -12,7 +19,24 @@
     gap: 30px;
 }
 
-.article-hero .hero-overlay  {
+div.hero-overlay-smaller-screen > a::before,
+.article-hero div.hero-overlay > a::before{
```

Comment:
please pull from `main`, "by " is already added in the DOM. So you can remove this.